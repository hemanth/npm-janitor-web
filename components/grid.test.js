import React from 'react'
import Grid from './grid'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <Grid>
      <div>1</div>
      <div>2</div>
      <div>3</div>
    </Grid>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
