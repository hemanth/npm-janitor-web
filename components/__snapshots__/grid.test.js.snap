// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="lIpXm"
>
  <div>
    1
  </div>
  <div>
    2
  </div>
  <div>
    3
  </div>
</div>
`;
